### PR TITLE
Adding LitmusChaos back to the overview page

### PR DESCRIPTION
**Pull request by aepfli**
Signed-off-by: Simon Schrottner <simon.schrottner@dynatrace.com>

### CODE REVIEW

**Comment by aepfli** [comment]
@StackScribe adding the tutorial back

**Comment by aepfli** [comment]
preview available here: https://deploy-preview-242--keptn-tutorials.netlify.app/

**Comment by aepfli** [comment]
This pullrequest exploded, with also fixing the tutorial overview - please have a look 

- https://tutorials.keptn.sh/
- https://tutorials.keptn.sh/tutorials/keptn-full-tour-dynatrace-012/

File: site/tutorials/keptn-litmus-018/codelab.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "environment": "web",
+  "format": "html",
+  "prefix": "https://storage.googleapis.com",
+  "mainga": "UA-133584243-1",
+  "updated": "2022-12-05T16:29:24+01:00",
+  "id": "keptn-litmus-018",
+  "duration": 44,
+  "title": "Resilience evaluation with LitmusChaos, Prometheus, and Keptn",
+  "authors": "Jürgen Etzlstorfer",
```
Comment (thisthat) [inline site/tutorials/keptn-litmus-018/codelab.json:10]: I think you can add your name here ;)

**Review by thisthat — APPROVED** [review]

File: site/tutorials/keptn-litmus-018/codelab.json (hunk 1 above)
Comment (aepfli) [reply site/tutorials/keptn-litmus-018/codelab.json]: changed

Comment (StackScribe) [reply site/tutorials/keptn-litmus-018/codelab.json]: I missed this!  Shouldn't this show Shubham Chaudhary and karthik_s -- don't have an actual name for the second name?  They are the ones who stepped up and did all the work!

**Review by StackScribe — APPROVED** [review]
lgtm, although you do know that this PR is a bit overloaded, right?  ;-)

File: site/tutorials/keptn-litmus-018/codelab.json (hunk 1 above)
Comment (aepfli) [reply site/tutorials/keptn-litmus-018/codelab.json]: did they rewrite it, or did they change the initial setup, I added shubham Chaudhary, but should i only write `karthik_s` or is there a full name available :)

**Pull request merged**